app/controllers/polls_controller.rb

```diff
@@ -30,7 +30,13 @@ def edit
 
   # POST /polls
   def create
-    @poll = Poll.new(poll_params)
+    current_poll_params = poll_params
```

Comment:
Here should be authorization added as well, but we handle this later. Especially to the fact that this is not your code ;)